LondonBritishLibrary/orient/BLorient587.xml

```diff
@@ -38,57 +38,68 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                            <notatedMusic><desc>With musical notes.</desc></notatedMusic>
+                            <incipit xml:lang="gez">ምዕራፍ፡ በኵሉ፡ መዋዕል፡ እምዮሐንስ፡ እስከ፡ ዮሐንስ፡</incipit>
+                        </msItem>
                         
                     </msContents>
                     
-                    
-                    
-                    
-                    
-                    
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
                                 </support>
                                 <extent>
-                                    <measure unit="leaf">90</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>225</height>
-                                        <width>184</width>
+                                    <measure unit="leaf">2+90+1</measure>
+                                    <note>Two unnumbered blank leaves at the beginning and one at the end.</note>
+                                    <dimensions type="outer" unit="in">
+                                        <height>8.88</height>
+                                        <width>7.25</width>
                                     </dimensions>
                                 </extent>
                                 
                                 
                             </supportDesc>
                             
                             <layoutDesc>
-                                <layout columns=" 3" writtenLines="23" cert="high"/>
+                                <layout columns=" 3" writtenLines="23"/>
                             </layoutDesc>
                         </objectDesc>
                         
                         <handDesc>
-                            
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is inelegantly written of the <date>19th century</date>
-                                </desc>
-                                
-                                <date notBefore="1800" notAfter="1900"/>
-                                
+                                <desc>Inelegantly written.</desc>
+                                <date notBefore="1800" notAfter="1868"/>
                             </handNote>
-                            
                         </handDesc>   
+                        
                         <additions>
-                            
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#67r"/>
+                                    <desc>A Patriarch John is mentioned in a prayer, maybe
```

Comment:
"A Patriarch John" looks a little strage (I understand what is meant, though), sure that this constellation is possible?

Reply:
If you mean English, the English is correct. means "A certain ...."

Reply:
"Patriarch John" is from Wright, the hypothesis from me, I will be happy for a better solution!

Reply:
I would propose a small change: "A patriarch John".